injective/assetlist.json

```diff
@@ -1323,6 +1323,42 @@
         "twitter": "https://x.com/neptune_finance"
       },
       "type_asset": "sdk.coin"
+    },
+    {
+      "description": "USDi token from Ethereum via Peggy bridge.",
+      "denom_units": [
+        {
+          "denom": "cw20:peggy0x83E7D0451da91Ac509cd7F545Fb4AA04D4dD3BA8",
+          "exponent": 0
+        },
+        {
+          "denom": "USDi",
+          "exponent": 18
+        }
+      ],
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "peggy0x83E7D0451da91Ac509cd7F545Fb4AA04D4dD3BA8"
```

Comment:
the word peggy should be removed from the denom reference since it's not the ethereum contract. just "0x83E7D0451da91Ac509cd7F545Fb4AA04D4dD3BA8"